README.md

```diff
@@ -78,7 +78,8 @@ or the configuration file can be specified by cli args as well.
 * `make run-mock-registry`: Mock registry. Entirely separate binary.
 * `make test`: Runs the test suite.
 * `make vendor`: Updates the dependencies
-* `make build`: Builds a docker container of the current source
+* `make dev-image`: Builds a docker container of the current source
+* `make release-image` Builds a docker container using the latest rpm from [Copr](https://copr.fedorainfracloud.org/coprs/g/ansible-service-broker/ansible-service-broker/)
```

Comment:
Talked to @jmrodi. Think we're going to do
make release-image
make push
make release: release-image

We may change `make release` to be `make release: release-image push` later but for now we'll leave them as discrete tasks.

Sounds like `make build` might get used for a go build and `make install` changed to install the broker in /usr/local/bin or something so it might be a good time to free up `make build` in case we make that change.